scripts/telnet-fingerprint.nse

```diff
@@ -0,0 +1,96 @@
+local stdnse = require "stdnse"
+local shortport = require "shortport"
+local comm = require "comm"
+local nmap = require "nmap"
+
+description = [[
+Attempts to fingerprint an open telnet service based on Telnet commands sent
+by the server upon connection. See RFC854 for more details.
+
+Original idea from telnetfp by Palmers of Team TESO.
+
+Please send new or incorrect fingerprint data to daniel@planethacker.net
```

Comment:
Avoid putting your email address here, leave a general dev@nmap.org or completely skip it.